src/Microsoft.DotNet.Interactive.Jupyter.Tests/CompleteRequestHandlerTests.cs

```diff
@@ -27,7 +29,7 @@ public CompleteRequestHandlerTests(ITestOutputHelper output) : base(output)
         public async Task send_completeReply_on_CompleteRequest()
         {
             var scheduler = CreateScheduler();
-            var request = ZeroMQMessage.Create(new CompleteRequest("System.Console.", 15));
+            var request = ZeroMQMessage.Create(new CompleteRequest("System.Console.", 15), new Session());
```

Comment:
This boilerplate change in all of the callers suggests this parameter should be nullable and coalesce to a default.